lib/retryable.rb

```diff
@@ -0,0 +1,17 @@
+module Retryable
```

Comment:
Faraday has a pretty sophisticated retry package: https://github.com/lostisland/faraday-retry

Reply:
Ah didn't know about this. I could go either way. This is a pretty simple implementation.

Unless that gem can effectively handle rate limiting out of the box the way we need it. In that case I'd probably go with the gem.

Reply:
The less you do, the less you have to maintain :)